Tools/Scripts/webkitpy/layout_tests/controllers/manager.py

```diff
@@ -726,14 +726,34 @@ def _worker_number(worker_name):
 
     def _print_expectation_line_for_test(self, format_string, test, device_type):
         test_path = test.test_path
-        line = self._expectations[device_type].model().get_expectation_line(test_path)
-        print(format_string.format(test_path,
-                                   line.expected_behavior,
-                                   self._expectations[device_type].readable_filename_and_line_number(line),
-                                   line.original_string or ''))
+        main_line = self._expectations[device_type].model().get_expectation_line(test_path)
+        if not self._options.verbose:
+            print(format_string.format(test_path,
+                                       "",
+                                       main_line.expected_behavior,
+                                       self._expectations[device_type].readable_filename_and_line_number(main_line),
+                                       main_line.original_string or ''))
+        else:
+            lines = self._expectations[device_type].model().get_expectation_lines(test_path)
+            before_main_line = True
+            decision = ""
+            for line in lines:
+                if line == main_line:
+                    decision = "-> "
+                    before_main_line = False
+                elif before_main_line:
+                    decision = "v  "
+                else:
+                    decision = "^  "
+                print(format_string.format(test_path,
+                                           decision,
+                                           line.expected_behavior,
+                                           self._expectations[device_type].readable_filename_and_line_number(line),
+                                           line.original_string or ''))
+                test_path = ' ' * len(test_path)
```

Comment:
That's to avoid repeating the same file over multiple lines.
Without replacement:
```
path v  skip
path -> pass
path ^  pass
```
With replacement:
```
path v  skip
     -> pass
     ^  pass
```
Should I add a comment, or do something else to make it clearer?

Reply:
I've decided to remove this little bit of cleverness. The output will be a bit busier with repeated filenames, but it's still readable, and more importantly it would survive grep-ing for subsets without losing context.